### PR TITLE
StoryLabels to react component

### PR DESCRIPTION
**Pull request by sergiomurilovarela**
* Create StoryLabels component
* Wrap Tagit into react component

### CODE REVIEW

File: app/assets/javascripts/components/jquery_wrappers/InputTagit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  loadTagit() {
+    $(this.input).tagit(
+      this.tagitProperties()
+    ).on('change', this.props.onChange);
+  }
+
+  tagitProperties() {
+    console.log(this.props);
```
Comment (romulostorel) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: debug

Comment (sergiomurilovarela) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: Fixed.

File: app/assets/javascripts/components/jquery_wrappers/InputTagit.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class InputTagit extends React.Component {
```
Comment (talyssonoc) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: What do you think about renaming it to `TaggedInput`? This way the name will be decoupled from `tagit` lib.

File: app/assets/javascripts/components/jquery_wrappers/InputTagit.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  tagitProperties() {
+    return {
+      availableTags: this.props.availableLabels,
+      readOnly: this.props.disabled,
+    };
+  }
+
+  render() {
+    return (
+      <input ref={ input => { this.input = input } } { ...this.props.input } tabIndex='this.props.labels' />
```
Comment (talyssonoc) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: Shouldn't the value of `tabIndex` be `{this.props.availableLabels}` (inside braces) instead the string `'this.props.labels'`?

Comment (talyssonoc) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: Also, can we break this line to make it a little shorter? You could indent it like this:

```jsx
<input
  ref={...}
  {...this.props.input}
   tabIndex={...}
/>
```

File: app/assets/javascripts/components/jquery_wrappers/InputTagit.js (hunk 2 above)
Comment (sergiomurilovarela) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: Sure.

File: app/assets/javascripts/components/jquery_wrappers/InputTagit.js (hunk 3 above)
Comment (sergiomurilovarela) [inline app/assets/javascripts/components/jquery_wrappers/InputTagit.js]: In fact, we don't need the tabIndex attribute. And the lines are broken now.

**Pull request merged**